internal/audit/local/badgerdb.go

```diff
@@ -402,6 +416,14 @@ func GenKey(prefix []byte, id audit.IDBytes) []byte {
 	return key[:]
 }
 
+func GenKeyWithByteSize(prefix []byte, id audit.IDBytes, nBytes int) []byte {
+	key := GenKey(prefix, id)
+	hex := fmt.Sprintf("%0*x", keyByteSizeLen, nBytes)
```

Comment:
I am curious why the size is encoded as hex. Couldn't we just store it encoded as Big Endian using [encoding/binary](https://pkg.go.dev/encoding/binary)?

Reply:
I suppose because when I initially implemented it, human-readability seemed like a good idea 😆. The `encoding` lib makes more sense 👍